python/lsst/ts/externalscripts/base_make_bias.py

```diff
@@ -0,0 +1,104 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["BaseMakeBias"]
+
+import yaml
+import abc
+
+from lsst.ts import salobj
+
+
+class BaseMakeBias(salobj.BaseScript, metaclass=abc.ABCMeta):
+    """ Base class for taking biases and construct a master bias.
+
+    Parameters
+    ----------
+    index : `int`
+        SAL index of this script
+    """
+
+    def __init__(self, index, descr):
+        super().__init__(index=index, descr=descr)
+
```

Comment:
You probably want to create an abstract property that returns the "camera", e.g.;

```
    @property
    @abc.abstractmethod
    def camera(self):
        raise NotImplementedError()
```

Then, on the child classes you would do:

```
def __init__(self, index, descr):
    ...
    self._comcam = ComCam(...)
    ...

@property
def camera(self):
    return self._comcam
```